modin/core/execution/ray/__init__.py

```diff
@@ -12,3 +12,21 @@
 # governing permissions and limitations under the License.
 
 """Modin's functionality related to Ray execution engine."""
+
+from .common import initialize_ray, RayTask, SignalActor
+from .generic import (
+    display_time_updates,
+    call_progress_bar,
+    GenericRayDataframePartitionManager,
+    RayIO,
+)
+
+__all__ = [
```

Comment:
Not sure we should expose all the objects from this file but I would like to listen to other thoughts. Probably I would keep this file empty.

Reply:
I added these objects to this file because they are used in other external modules, thus making the interface more explicit. (Except `display_time_updates`, `call_progress_bar` functions; I can remove they from the file)

Reply:
Let's leave this file empty.

Reply:
By defining an interface in this file, we encapsulate the structure of this module from external code. Why are you suggesting leaving it empty?

Reply:
I prefer things bound to their modules like PartitionManager should be accessed from partitioning module, but not from a higher one.

Reply:
In this case, with any changes in the path to the module with the PartitionManager, you will have to change imports not only inside the module, but also outside.

Reply:
That's true but I hope we got the stable structure for core directory.

Reply:
We can only hope for this, however, with the proposed approach, there is no need to even hope :)
Using this approach is very convenient, for example, to quickly see the external links of the module, what simplifies architectural analysis (at the component level).

Reply:
I still tend to stick my point. Let's listen to other opinions.

Reply:
cc @modin-project/modin-core

Reply:
I would side with @YarShev here because this is how we do for all other places.

It is really convenient to keep subpackage details easy to refactor, so if this would be a way to go if we were implementing the things from scratch.